keps/sig-instrumentation/20190131-new-event-api-ga-graduation.md

```diff
@@ -1,3 +1,30 @@
+---
```

Comment:
Given you touch this file, let's stick to the new kep format and create the kep.yaml with metadata and readme with the actual content.

Reply:
Done.